htsohm/utilities.py

```diff
@@ -71,8 +71,12 @@ def read_config_file(run_id):
 
 def evaluate_convergence(run_id):
     '''Counts number of materials in each bin and returns variance of these counts.'''
-    bin_counts = session.query(func.count(Material.id)).filter(Material.run_id == run_id).group_by(
-        Material.methane_loading_bin, Material.surface_area_bin, Material.void_fraction_bin
+    bin_counts = session \
+        .query(
+            func.count(Material.id)).filter(Material.run_id == run_id
```

Comment:
.filter should be at the same level as .query:

.query(func.count(Material.id))
.filter(Material.run_id == run_id)